Mathlib/Analysis/Analytic/IsolatedZeros.lean

```diff
@@ -133,6 +133,15 @@ theorem eventually_eq_or_eventually_ne (hf : AnalyticAt 𝕜 f z₀) (hg : Analy
     (∀ᶠ z in 𝓝 z₀, f z = g z) ∨ ∀ᶠ z in 𝓝[≠] z₀, f z ≠ g z := by
   simpa [sub_eq_zero] using (hf.sub hg).eventually_eq_zero_or_eventually_ne_zero
 
+/-- Two analytic functions agree on a punctured neighborhood iff they agree on a neighborhood. -/
+theorem eventuallyEq_nhd_iff_eventuallyEq_nhdNE (hf : AnalyticAt 𝕜 f z₀) (hg : AnalyticAt 𝕜 g z₀) :
```

Comment:
I think this name should swap its components, since the NE is on the LHS of the iff.

Reply:
Upsetting. Fixed.